### PR TITLE
Improving our gulp file

### PR DESCRIPTION
**Pull request by bjedrzejewski**

### CODE REVIEW

File: gulp-file.ts [hunk 1]
```diff
@@ -109,18 +121,19 @@ gulp.task('min', ['concat'], () => {
...
-    let gls = require('gulp-live-server');
+    let server = createServer();
+    server.start();
+});
 
-    let portNumber = 8080;
-    let server = gls.static(paths.site.app, portNumber);
+gulp.task('app:connect', [], () => {
+    let server = createServer();
     server.start();
```
Comment (owennw) [inline gulp-file.ts]: You don't need the empty array here. Also I don't know if we like a style of `createServer().start()`? Some people don't like following `()` with `.`s. How do we feel?

Comment (bjedrzejewski) [inline gulp-file.ts]: I don't mind it really. Can make it into one liner. I will also remove the array.

File: gulp-file.ts [hunk 2]
```diff
@@ -87,7 +95,11 @@ gulp.task('build:src', ['lint', 'clean'], () => {
     return build(files.src.ts, paths.src.js);
 });
 
-gulp.task('build', ['build:tests', 'build:src']);
+gulp.task('build:appsrc', ['lint', 'clean'], () => {
+    return build(files.site.ts, paths.site.app);
+});
+
```
Comment (owennw) [inline gulp-file.ts]: I'm not sure if this is where we want to be building all of our appsrc files. In my mind the ./app directory would contain the HTML, techstudio.js(.min, .map) and the app.js(.min, .map) and that's it. The intermediate appsrc .js files might belong elsewhere? But I don't know where..

**Comment by owennw** [comment]
Just a few tweaks, and a question of where do we build the appsrc files?

File: gulp-file.ts (hunk 1 above)
Comment (owennw) [inline gulp-file.ts]: Actually. given that function is being used twice and both times the server is being started maybe the `.start()` portion belongs in `createServer`, as a `startServer()` method. What do you think?

**Comment by bjedrzejewski** [comment]
I dont know yet. Let me implement the refactoring and try to build something. I think it may be easier to figure it out once there is some angular2 there.

File: gulp-file.ts [hunk 3]
```diff
@@ -46,12 +46,21 @@ let files = {
...
 };
 
+let createServer = () => {
+    let gls = require('gulp-live-server');
+
+    let portNumber = 8080;
+    return gls.static(paths.site.app, portNumber);
+};
+
+
```
Comment (owennw) [inline gulp-file.ts]: Ah the ol' double carriage return between methods!

**Pull request merged**